backend/pom.xml

```diff
@@ -38,12 +38,25 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.loki4j</groupId>
+            <artifactId>loki-logback-appender</artifactId>
+            <version>1.6.0</version>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
```

Comment:
Is this dependency needed for logging or metrics?

Reply:
> Is this dependency needed for logging or metrics?

No, its not needed at all. It just helps me trigger my endpoints points easily in swagger ui